src/meson.build

```diff
@@ -43,6 +43,4 @@ kiwixlib = library('kiwix',
                    include_directories : inc,
                    dependencies : all_deps,
                    version: meson.project_version(),
-                   install: true,
-                   install_dir: install_dir,
-                   install_rpath: '$ORIGIN')
+                   install: true)
```

Comment:
You also remove the `install_dir` option.

This is needed for android installation.

Reply:
Yes. Restored.